styles/index.css

```diff
@@ -0,0 +1,428 @@
+* {
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif
+}
+
+/* skeleton */
+header {
+  clear: both;
+  margin-bottom: 0;
+}
+
+main {
+  margin-left: 15rem;
```

Comment:
this is too large and makes your site pretty squished. I'd take it out entirely.